.github/workflows/test-changed.yml

```diff
@@ -78,3 +78,41 @@ jobs:
         run: xvfb-run yarn test:changed core
         env:
           BROWSERS: 'Firefox'
+          
+          
+  test-webkit:
+    name: Test Packages With Changed Files in Webkit
+    runs-on: macos-latest
+
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Set up Node (20)
+        uses: actions/setup-node@v3
+        with:
+          node-version: 20.x
+      - name: Test setup and yarn install
+        run: |
+          cp config/ci.config.json config/project.json
+          yarn
+          npx playwright install webkit
+      - name: build
+        run: yarn build:changed core
+      - name: Run tests on changed packages
+        run: |
+          npx lerna run test:browser --scope @firebase/database-compat
```

Comment:
?

Reply:
I made this change to help me identify the cause of the flakes. I've reverted all these changes.